src/services/saveImagesToUSB.ts

```diff
@@ -0,0 +1,54 @@
+import { IService } from '../types';
+import { DiskUsage } from 'types/motionModel';
+import { readdirSync, existsSync, mkdirSync, rename } from 'fs';
+import { USB_WRITE_PATH } from 'config';
+import { getDateFromUnicodeTimestamp } from '../util';
+
+const diskUsage: DiskUsage = {};
+
+export const getDiskUsage = () => {
+    return diskUsage;
+};
+
+export const SaveImagesToUSB: IService = {
+    execute: async () => {
+        try {
+            const directoryPath = USB_WRITE_PATH;
+            const files = readdirSync(directoryPath);
```

Comment:
I need to double-confirm that if this request is very long (for example, if there're lots of images in directory), it will not hang the entire ODC API.
And we can test it the following way:
- Let your camera to create a looooot of frames on your USB stick, just run the camera and let it create thousands of them, without moving to any folders
- Then, update dashcam-api.js on the camera with your new solution, and try to execute it, so it will try to move all those thousands of images to folders
- Check that API itself is alive and working during the execution of your heavy request (you can check if http://192.168.0.10:5000/api/1/info is still responsive)
- If API is alive, and your script is capable of moving thousands of images to sub-folders, then we're ready to go